front-end/src/App.js

```diff
@@ -1,23 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import Container from '@material-ui/core/Container';
 import Box from '@material-ui/core/Box';
 import { Route, HashRouter } from 'react-router-dom';
 import NavBar from './components/NavBar';
 import About from './components/About';
 import SignIn from './components/SignIn';
 import User from './components/User';
+import Admin from './components/Admin';
+import GuardedRoute from './GuardedRoute';
+
+const App = () => {
+  const [isAuthenticated, setisAuthenticated] = useState(false);
+  const [signInAttempt, setsignInAttempt] = useState(false);
```

Comment:
Naive question: is it standard to not have the setter in full camel-case? i.e. `setsignInAttempt` instead of `setSignInAttempt`? Just curious, I'm not very familiar with `useState()`.

Reply:
So it seems like in the react documentation we should still use camel-case. I can make the change real quick. Good call out!